src/ajax.js

```diff
@@ -13,6 +13,7 @@ define([
 
 var
 	rhash = /#.*$/,
+	rextraQuestions = /\?+/,
```

Comment:
Ah, I see, you're changing the first question mark only and that's why it needs to be one. Seems fine as it is then.